### PR TITLE
Make HTML amplicon plot

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

**Comment by jkokosar** [comment]
Depends on https://github.com/genialis/resolwe-bio/pull/369

**Comment by mstajdohar** [comment]
@JureZmrzlikar This should pass now that the `dnaseq` docker image was built on the Dockerhub. Please rebase.

**Comment by mstajdohar** [comment]
I have also tagged the image [(1.0.0)](https://hub.docker.com/r/resolwebio/dnaseq/tags/) so I suggest you reference the tag in processes.

**Comment by JureZmrzlikar** [comment]
Rebased and tagged dnaseq iage with 1.0.0. Note that @jkokosar wanted to test this locally before merging. If this is still the case, wait with merging until then...

File: resolwe_bio/processes/exome_coverage/coveragebed.yml [hunk 1]
```diff
@@ -9,9 +9,9 @@
     expression-engine: jinja
     executor:
       docker:
-        image: resolwebio/legacy:latest
+        image: resolwebio/dnaseq:1.0.0
   data_name: "Coverage ({{ alignment|sample_name|default('?') }})"
-  version: 2.2.2
+  version: 2.2.3
```
Comment (jkokosar) [inline resolwe_bio/processes/exome_coverage/coveragebed.yml]: Bump this to version 3.0.0

**Comment by JureZmrzlikar** [comment]
Done.

**Pull request merged**